src/common/logger.js

```diff
@@ -0,0 +1,23 @@
+const pino = require("pino");
```

Comment:
Please check usages of quotation marks - it's better to use them in one style - only '' or only "" - in all project